packages/default-theme/src/components/forms/BillingAddressUserForm.vue

```diff
@@ -43,25 +43,20 @@
   </div>
 </template>
 <script>
-import {
-  SfModal,
-  SfList,
-  SfRadio,
-  SfCheckbox,
-  SfAddressPicker,
-} from "@storefront-ui/vue"
+import { SfModal, SfList, SfRadio, SfAddressPicker } from "@storefront-ui/vue"
 import { useSessionContext, useUser } from "@shopware-pwa/composables"
 import SwButton from "@/components/atoms/SwButton.vue"
 import { ref, watch } from "@vue/composition-api"
 import SwAddressForm from "@/components/forms/SwAddressForm.vue"
+// import SwCheckbox from '@/components/atoms/SwCheckbox.vue'
```

Comment:
temporary commented because it's not used - 👌🏻

Reply:
exactly :)